### DISCUSSION TITLE
Disable frame limiter option when using a vsync-locked renderer (Metal)

### DISCUSSION THREAD

**Discussion by brokenparts** [ppy/osu#30396] [Ideas] [answered]
Hi. Since the Metal renderer is always synced to the display refresh rate, I am suggesting that the frame limiter option be disabled when this backend is selected. Right now it just ignores whatever the user has set. This is confusing and looks like a bug without any explanation.

If visually disabling the dropdown doesn't make sense, maybe some explanatory text could be added.

False-positive reports: #24024 #25985 #30220 #30302

**Comment by peppy** [comment]
Sounds valid.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/issues/30397
